### DISCUSSION TITLE
Add Default Prompt Popup

### DISCUSSION THREAD

**Discussion by Sagar27888** [guangzhengli/ChatFiles#53] [Ideas]
Hello,

I have seen your repo and it looks awesome. However, I have suggested one feature if you add on in project like once you upload any file default prompt messages appear means if any non-technical person don't know about how to ask questions which can help to select in default prompt popup.

**Comment by guangzhengli** [comment]
Thank you for your suggestion. We have already planned to implement automatic prompts for users after uploading files. If anyone has any good suggestions regarding these prompts, please reply here. 

Thank you very much and best wishes.

**Reply by Sagar27888** [reply]
Thanks for your prompt response... I would suggest we can provide some category like Insurance, Legal Advice etc... and once user select any category we will display default prompt questions related to category which can help users to select questions without thinking more business engineering of prompt questions
